wls-common/swagger/src/test/java/de/muenchen/oss/wahllokalsystem/wls/common/swagger/SwaggerDefaultConfigurationTest.java

```diff
@@ -0,0 +1,75 @@
+package de.muenchen.oss.wahllokalsystem.wls.common.swagger;
+
+import io.swagger.v3.oas.models.Components;
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.Operation;
+import io.swagger.v3.oas.models.PathItem;
+import io.swagger.v3.oas.models.Paths;
+import io.swagger.v3.oas.models.responses.ApiResponse;
+import io.swagger.v3.oas.models.responses.ApiResponses;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springdoc.core.customizers.GlobalOpenApiCustomizer;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class SwaggerDefaultConfigurationTest {
+    private GlobalOpenApiCustomizer customizer;
+    private Components components;
+
+    @BeforeEach
+    void setUp() {
+        SwaggerDefaultConfiguration configuration = new SwaggerDefaultConfiguration();
+        customizer = configuration.errorCustomizer();
+        components = new Components();
+    }
+
+    @Test
+    public void should_AddErrorResponsesToGetOperation_when_CustomizerIsCalled() {
```

Comment:
Du könntest auch die ArchUnit-Tests für die Namingconventions auf dieses Modul loslassen (RULE_TEST_NAMING_CONVENTION_SHOULD_WHEN_MATCHED, RULE_BEFORE_EACH_NAMING_CONVENTION_MATCHED und RULE_AFTER_EACH_NAMING_CONVENTION_MATCHED aus MethodRules)

```suggestion
    public void should_addErrorResponsesToGetOperation_when_customizerIsCalled() {
```